scripts/build_torch_wheels.sh

```diff
@@ -67,7 +67,8 @@ function build_and_install_torch() {
   # Only checkout dependencies once PT commit ID checked out.
   git submodule update --init --recursive
   # Apply patches to PT which are required by the XLA support.
-  xla/scripts/apply_patches.sh
+  # xla/scripts/apply_patches.sh
```

Comment:
Why did we comment this out?

Reply:
Cannot be commented out.

Reply:
I could remove the commented line. script is executed in the next line.

Reply:
to clarify; apply_patches.sh is executed in line 71. just not via a relative path.

Reply:
Now that I see that ... why that weird sed replace?
Can't you just append the patch script name to a `dirname $0`?

Reply:
sure, changing to dirname

Reply:
and removing comment.